src/main/java/com/denizenscript/depenizen/bukkit/objects/luckperms/LuckPermsGroupTag.java

```diff
@@ -0,0 +1,150 @@
+package com.denizenscript.depenizen.bukkit.objects.luckperms;
+
+import com.denizenscript.denizencore.objects.Fetchable;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.tags.Attribute;
+import com.denizenscript.denizencore.tags.ObjectTagProcessor;
+import com.denizenscript.denizencore.tags.TagContext;
+import com.denizenscript.depenizen.bukkit.bridges.LuckPermsBridge;
+import net.luckperms.api.model.group.Group;
+
+import java.util.OptionalInt;
+
+public class LuckPermsGroupTag implements ObjectTag {
+
+    public static ObjectTagProcessor<LuckPermsGroupTag> tagProcessor = new ObjectTagProcessor<>();
+
+    // <--[ObjectType]
+    // @name LuckPermsGroupTag
+    // @prefix luckpermsgroup
+    // @base ElementTag
+    // @format
+    // The identity format for group is <group_name>
+    // For example, 'luckpermsgroup@my_group'.
+    //
+    // @plugin Depenizen, LuckPerms
+    // @description
+    // A LuckPermsGroupTag represents a LuckPerms group.
+    //
+    // -->
+
+    @Fetchable("luckpermsgroup")
+    public static LuckPermsGroupTag valueOf(String string, TagContext context) {
+        if (string == null) {
+            return null;
+        }
+        string = string.replace("luckpermsgroup@", "");
+        Group group = LuckPermsBridge.luckPermsInstance.getGroupManager().getGroup(string);
+        if (group == null) {
+            return null;
+        }
+        return new LuckPermsGroupTag(group);
+    }
+
+    public static boolean matches(String arg) {
+        arg = arg.replace("luckpermsgroup@", "");
```

Comment:
Same here - `matches` method generally just return `true` if it starts with the object's prefix though